test/transcoding/SPV_KHR_integer_dot_product_OCLtoSPIRV_char4.ll

```diff
@@ -0,0 +1,85 @@
+; RUN: llvm-as < %s -o %t.bc
+; RUN: llvm-spirv -s %t.bc -o %t.regularized.bc
```

Comment:
You will need to add `--spirv-ext=+SPV_KHR_integer_dot_product` when converting a `.bc` that will be using the extension.